ruamel_yaml/prepare.bash

```diff
@@ -25,4 +25,4 @@ touch $SRC_DIR/ruamel_yaml/ext/__init__.py
 
 cp $RECIPE_DIR/setup.py $SRC_DIR/
 cp $RECIPE_DIR/__init__.py $SRC_DIR/ruamel_yaml/
-sed -i -e 's/__version__.*/__version__ = "$VERSION"/' $SRC_DIR/ruamel_yaml/__init__.py
+sed -i -e "s/__version__.*/__version__ = "'"'"${VERSION}"'"'"/" $SRC_DIR/ruamel_yaml/__init__.py
```

Comment:
As single quotes were in use already, we are unable to inject the version from the environment variable `VERSION`. Instead we just literally write `$VERSION`. Here we switch to double quotes so that is not an issue and escape them so that the `$VERSION` can be written into a Python string.